examples/anonymous-functions/anonymous_functions.metatags

```diff
@@ -0,0 +1,2 @@
+description: Anonymous functions in Ballerina are a syntactic shortcut for defining inline functions. It only differs from a normal function because it does not have a name.
+keywords: ballerina, ballerina by example, bbe, anonymous functions
```

Comment:
Should we consider adding "lambda functions" to this keywords list? @MaryamZi @hasithaa

Reply:
+1 for lambda, I don't think I would ever search `anonymous function` in a google seach.

Reply:
+1 to add "lambda functions" also.

Reply:
Additionally, can we rename this BBE to "lambda functions" @lafernando @lakwarus WDYT?

Reply:
Btw Spec use word "Anonymous function", instead of "Lambda"

Reply:
+1 for keeping both

Reply:
I think we better keep the name of the BBE as "Anonymous functions". But for purposes like this, I think we can use the term lambda function since it's a commonly used name for anonymous functions.